fe/fe-core/src/main/java/org/apache/doris/journal/JournalEntity.java

```diff
@@ -896,6 +897,11 @@ public void readFields(DataInput in) throws IOException {
                 isRead = true;
                 break;
             }
+            case OperationType.OP_DELETE_TABLE_STATS: {
```

Comment:
do we need add this operationtype to doris-2.0 branch?

Reply:
I think we need